yang/frr-isisd.yang

```diff
@@ -341,8 +347,7 @@ module frr-isisd {
     }
 
     leaf vrf {
-      type string;
-      default "default";
```

Comment:
I think you removed this default value by accident. Bringing it back should solve the missing callback problem.

> I didn't look deep at it yet, but I wonder why isisd has a separate "vrf" leaf in the "interface-config" grouping and don't just use the "vrf" leaf of the interface it's augmenting.

That was necessary to work around a problem related to per-daemon configuration files that are parsed before isisd receives interface information from zebra. Long term we should try to get rid of that leaf as it's redundant and confusing.

Reply:
Thanks for pointing out and for the explanation. Default value is returned.